charm/kafka/templates/server.properties

```diff
@@ -28,12 +28,20 @@ broker.id={{ broker_id }}
 #     listeners = listener_name://host_name:port
 #   EXAMPLE:
 #     listeners = PLAINTEXT://your.host.name:9092
-listeners=PLAINTEXT://{{ bind_addr }}:{{ port }}
+listeners=SSL://{{ bind_addr }}:{{ port }}
+
+ssl.truststore.location={{ ca_keystore }}
+ssl.truststore.password={{ keystore_password }}
+ssl.keystore.location={{ server_keystore }}
+ssl.keystore.password={{ keystore_password }}
+ssl.key.password={{ keystore_password }}
+security.inter.broker.protocol=SSL
```

Comment:
Do the brokers in a cluster need to issue client certs in order to be able to talk to each other?

Reply:
no.. easyrsa will issue client certs